src/framework/theme/components/select/option.component.ts

```diff
@@ -42,6 +42,9 @@ import { NbSelectComponent } from './select.component';
   `,
 })
 export class NbOptionComponent<T> implements OnDestroy {
+
+  protected disabledByGroup: boolean = false;
```

Comment:
We don't need to specify `: boolean` explicitly here. Because ts compiler can infer type when you assign value in declaration. Also as it's quite obvious that this value always will be `boolean` 😄 
As I remember we have the following `tslint` rule.